app/services/chatbot/chatbot-interfaces.ts

```diff
@@ -0,0 +1,146 @@
+// state
+export interface ChatbotApiServiceState {
+  api_token: string;
+  socket_token: string;
+}
+
+// responses
+export interface ChatbotAPIPostResponse {
+  success: boolean;
+}
+
+export interface ChatbotAPIPutResponse {
+  success: boolean;
+}
+
+
+export interface DafaultCommandsResponse {
+  commands: DafaultCommandsSlug;
+  'link-protection': DafaultCommandsSlug;
```

Comment:
Lol why is this not `snake_case` like every other API response attribute.  Maybe something to mention to Marcin.